client-tez/pom.xml

```diff
@@ -26,61 +27,81 @@
         <relativePath>../pom.xml</relativePath>
     </parent>
 
-    <groupId>org.apache.uniffle</groupId>
```

Comment:
Why do we remove this groupId? Could we keep consistent style with other submodule pom files?

Reply:
ok, we reverted it.